concepts/pipelines/index.md

```diff
@@ -26,3 +30,46 @@ Automation does not end with orchestrating only the high-level steps of your rel
 * Each given step is actually a set of tasks that need polling, remediation to ensure requisite state is reached prior to proceeding
 
 * A given task often entails multiple API calls to the specific cloud platform, cognizant of expected response codes and remediating actions in failure
+
+## Pipeline templates
+
+Managed pipeline templates are a way to scalably manage pipelines across different teams. 
+
+Pipeline templates have two main components:
+
+* Template
+
+  A pipeline template defines a parameterized pipeline, with the set of
+  variables for which your users will provide values.
+
+* Configuration
+
+  A concrete implementation of a template. Configurations can inject new stages
+  into the final pipeline graph and inherit or override, or both, triggers,
+  notifications, and parameters.
+
+
+### Terminology
+
+#### Pipeline template
+
+A parameterized pipeline, minus the pipeline configuration found on a pipeline
+instance. Based on this template, your Spinnaker users can more easily create
```

Comment:
"Spinnaker users" terminology might change in the future to be consistent with the Declarative verbiage.